packages/ses/src/compartment.js

```diff
@@ -297,6 +297,26 @@ export const makeCompartmentConstructor = (
 
     assign(globalObject, endowments);
 
+    /**
+     * @param {string} fullSpecifier
+     */
```

Comment:
- What is a `fullSpecifier`? 
- Please add description of function
- Please add return type of function

Reply:
Answering in comment, but for you:
- A *full specifier* is a key in the compartment's memo of known modules. When we add support for import attributes (`import ansiSys from 'autoexec.bat' with { type: 'bmp' }`), the full specifier will be a _part_ of the compartment memo key. Functions like `compartment.import` take full specifiers. In Node.js and the web, the full specifier happens to be a fully qualified URL, so `import.meta.url` is the same. In Compartment, the location and full specifier are not necessarily the same. With Compartment Mapper, the full specifier is the specifier you would see in a `package.json`, like `./src/compartment.js` (for internal linkage)` or `ses` (for external linkage).
- an *import specifier* by contrast is a specifier in the context of the surrounding module’s full specifier, like `./module-instance.js` is an import specifier in `./src/compartment.js` that resolves to a full specifier: `./src/module-instance.js`.
- you will note that a *relative specifier* like `./foo.js` can has different implications for full specifiers and import specifiers. The `./` prefix implies “internal” for a full specifier and full specifiers cannot have `..` path components. The `./` prefix for an import specifier implies that it should be resolved relative to the importer’s full specifier and `..` path components should eat the preceding path component of the full specifier.
- I don’t have active recall for the terms Node.js chose to distinguish these.